src/util/index.ts

```diff
@@ -128,6 +130,42 @@ export const toHashTable = (list: Array<any>, key: string) => {
   }, {});
 };
 
+export const fileToBase64 = (file: File): Promise<string | null> =>
```

Comment:
may you create a class with static methods like 
```
export class IpfsStorage {
  static fileToBase64() ...
}
```

Reply:
it can be a separated file

Reply:
Removed to separate file